sources/packages/web/src/constants/routes/RouteConstants.ts

```diff
@@ -22,6 +22,7 @@ export const StudentRoutesConst = {
   NOTIFICATIONS_SETTINGS: Symbol(),
   STUDENT_APPLICATION_SUMMARY: Symbol(),
   STUDENT_APPLICATION_DETAILS: Symbol(),
+  STUDENT_REQUEST_CHANGE: Symbol(),
```

Comment:
We need to decide if internally we would call this a "Request a Change" or Appeal.
From the DB perspective, we called everything an Appeal, so I believe that we can stick with it.
But I do understand that the UI is calling it "Student Request a Change", so on the Vue application maybe we can it as it is. Open for thoughts 😉

Reply:
Yes on the UI perspective I have kept the context as request for change. And in API perspective it is been kept as appeal. That seems to be ok IMO.